### PR TITLE
rename lookup8 and activate pass

### PR DESCRIPTION
**Pull request by aranXmos**

### CODE REVIEW

File: experimental/xformer/Transforms/ApplyXCPatterns.cpp [hunk 1]
```diff
@@ -83,17 +83,17 @@ DenseElementsAttr getLookupTable(PatternRewriter &rewriter, Operation *op) {
       op->getOperand(0).getType().dyn_cast<RankedTensorType>();
   auto inputQType =
       inputType.getElementType().dyn_cast<mlir::quant::UniformQuantizedType>();
-  auto inputScale = inputQType.getScale();
-  auto inputZeroPoint = inputQType.getZeroPoint();
+  double inputScale = inputQType.getScale();
```
Comment (ghost) [inline experimental/xformer/Transforms/ApplyXCPatterns.cpp:86]: Do we need to change these from auto?

File: experimental/xformer/Transforms/XCPatterns.td [hunk 2]
```diff
@@ -14,14 +14,14 @@ def getLookupTable
     : NativeCodeCall<"getLookupTable($_builder, $0.getDefiningOp())">;
 
 // TODO: DPK
```
Comment (panickal-xmos) [inline experimental/xformer/Transforms/XCPatterns.td:16]: Could you please remove the // TODO too? The activation work was the reason for the TODO. Thanks!

Comment (ghost) [inline experimental/xformer/Transforms/XCPatterns.td]: Could you please remove the // TODO too? The activation work was the reason for the TODO. Thanks!

File: experimental/xformer/Transforms/ApplyXCPatterns.cpp (hunk 1 above)
Comment (aranXmos) [reply experimental/xformer/Transforms/ApplyXCPatterns.cpp:86]: no, but I found it confusing while debugging and I think it improves readability?

Comment (ghost) [reply experimental/xformer/Transforms/ApplyXCPatterns.cpp:86]: Alright, I don't mind it either way.

**Review by ghost — APPROVED** [review]
+1

**Pull request merged**